examples/basic/src/index.js

```diff
@@ -2,16 +2,24 @@ import express from 'express';
 
 let app = require('./server').default;
 
+let green;
+
+try {
+  green = require('chalk').green;
+} catch (error) {
+  green = (msg) => msg;
+}
+
 if (module.hot) {
   module.hot.accept('./server', function() {
-    console.log('🔁  HMR Reloading `./server`...');
+    console.log(green('🔁 HMR Reloading `./server`...'));
```

Comment:
this is not necessary but if you think it's good to have it in green color, we have to update all the examples ...